### PR TITLE
added animedaisuki

### PR DESCRIPTION
**Pull request by Blatzar**
openload and streamango are dead it seems so I've 'disabled' them.

### CODE REVIEW

File: anime_downloader/sites/animedaisuki.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+            for a in soup: #removes capcha from link
+                if re.search(captcha_regex,a.get('href')):
+                    links.append(re.search(captcha_regex,a.get('href')).group(1))
+                else:
+                    links.append(a.get('href'))
+            
+            logger.debug(links)
+
+            for a in servers:
```
Comment (vn-ki) [inline anime_downloader/sites/animedaisuki.py:55]: Can you actually add this logic to the base class in a later PR? All your providers seem to use something like this.

File: anime_downloader/sites/animedaisuki.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                    title=a.select('h3')[0].text,
+                    url='https://animedaisuki.moe' + a.get('href'))
+                for a in search_results
+            ]
+            return(search_results)
+
+        def _scrape_episodes(self):
+            soup = helpers.soupify(helpers.get(self.url))
+            elements = soup.select('li.fa-play-circle > a')[::-1]
+            print(['https://animedaisuki.moe' + a.get('href') for a in elements if a.get('href').startswith('/watch/')])
```
Comment (vn-ki) [inline anime_downloader/sites/animedaisuki.py:26]: don't print

Comment (Blatzar) [reply anime_downloader/sites/animedaisuki.py:26]: oopsie, accidentally forgot to remove

File: anime_downloader/sites/animedaisuki.py (hunk 1 above)
Comment (Blatzar) [reply anime_downloader/sites/animedaisuki.py:55]: Good idea! Sure will do

**Comment by Blatzar** [comment]
I'm leaving the merge to you as I clearly misunderstood your opinion the last time regarding nyaa

**Comment by Blatzar** [comment]
Regarding the base class for getting the correct server based on config I was thinking something like this:
(reworking anime.py get_data)
```
    def get_data(self):
        self._sources = self._get_sources()

        if type(self._sources[0][0]) != str:

            version = self.config.get('version','subbed') #TODO add a flag for this
            servers = self.config.get('servers',[''])
            data = self._sources[0]
            sources = ''

            logger.debug(data)
            

            for a in servers: #trying all supported servers in order using the correct language
                for b in data:
                    if b['version'] == version:
                        if b['config'] == a:
                            if get_extractor(b['extractor']) == None:
                                continue
                            sources = [(b['extractor'], b['url'],)]
            
            logger.debug('No servers found in selected language. Trying all supported servers')
            
            if not sources:
                for a in servers: #trying all supported servers in order
                    for b in data:
                        if b['config'] == a:
                            if get_extractor(b['extractor']) == None:
                                continue
                            sources = [(b['extractor'], b['url'],)]            
            self._sources = sources

        logger.debug('Sources : {}'.format(self._sources))
```
This way it's still backward compatible (which we later can deprecate) and supports both language and priority of servers through config. For reference, data variable would look something like this:
```
[{'extractor': 'no_extractor', 'url': 'https://animedaisuki.moe/f/Yakusoku no Neverland - 01 [720p]-808d589c.mp4', 'config': 'official', 'version': 'subbed'}, {'extractor': 'no_extractor', 'url': 'https://animedaisuki.moe/f/Yakusoku no Neverland - 01 [720p]-808d589c.mp4', 'config': 'official', 'version': 'subbed'}]
```

**Pull request merged**